whatwg.org/.htaccess

```diff
@@ -107,7 +106,7 @@ redirect 301 /specs/web-apps/current-work/websrt.html https://w3c.github.io/webv
 redirect 301 /specs/web-apps/current-work/webvtt.html https://w3c.github.io/webvtt/
 redirect 301 /specs/web-apps/current-work/ https://html.spec.whatwg.org/
 redirect 301 /specs/web-forms/tests https://github.com/w3c/web-platform-tests
-redirect 301 /specs/web-workers/current-work/index https://html.spec.whatwg.org/multipage/workers.html
```

Comment:
Why change this?

Reply:
We don't have a redirect for `/specs/web-workers/current-work/`?

Reply:
Ah, I guess the idea is that nobody linked to /index to begin with, but we redirect that way to avoid redirecting the entire directory all the time.

Reply:
That is rather weird syntax-wise.

Reply:
Yes, it's pretty weird. I mean to make it less weird in the nginx equivalent, by excluding just rationale.html and redirecting everything else.

Reply:
I somewhat prefer it if we only redirect what we know to have been a URL that was not a 404, but I realize that's not how most redirect instructions work.

Reply:
It's very easy to do just that. What's not easy is to know what URLs have existed and linked to under a path.